### PR TITLE
XCMS 1.46.0 -> 1.50.1

### PR DESCRIPTION
**Pull request by lecorguille**
https://github.com/workflow4metabolomics/xcms/issues/34

### CODE REVIEW

File: galaxy/xcms_fillpeaks/abims_xcms_fillPeaks.xml [hunk 1]
```diff
@@ -268,6 +268,10 @@ Output files
 Changelog/News
 --------------
 
+**Version 2.2.0-1.50.1 - 23/03/2017**
```
Comment (bgruening) [inline galaxy/xcms_fillpeaks/abims_xcms_fillPeaks.xml]: I would not put this in a tool. Is this interesting for a user?

File: galaxy/xcms_retcor/abims_xcms_retcor.xml [hunk 2]
```diff
@@ -319,6 +319,10 @@ Output files
 Changelog/News
 --------------
 
+**Version 2.2.0-1.50.1 - 23/03/2017**
+
```
Comment (bgruening) [inline galaxy/xcms_retcor/abims_xcms_retcor.xml]: If you need it in the tool, it could be a macro.

File: galaxy/xcms_fillpeaks/abims_xcms_fillPeaks.xml (hunk 1 above)
Comment (lecorguille) [inline galaxy/xcms_fillpeaks/abims_xcms_fillPeaks.xml]: I think so.
It can be see as:
- a warning "Be careful, we have change something behind even if you may not see the difference"
- an ad, "Hey, look at that, we are still working on our wrapper, and follow the XMCS developments"

We can also adopt the `USER VISIBLE CHANGES:` tag ?

File: galaxy/xcms_retcor/abims_xcms_retcor.xml (hunk 2 above)
Comment (lecorguille) [inline galaxy/xcms_retcor/abims_xcms_retcor.xml]: Macro are sometime rather hard to follow.
And that way, it's easier for me to copy/paste in the README.md shiped with the tool for the TS and GitHub

**Comment by lecorguille** [comment]
Abort
See: https://github.com/workflow4metabolomics/xcms/issues/77
